muskathlon/models/event_compassion.py

```diff
@@ -8,19 +8,63 @@
 #    The licence is in the file __manifest__.py
 #
 ##############################################################################
-from odoo import models, fields
+from odoo import models, fields, api
+import re
 
 
 class EventCompassion(models.Model):
     _inherit = 'crm.event.compassion'
 
     thank_you_text = fields.Html(translate=True)
-
     muskathlon_event_id = fields.Char(
         string="Muskathlon event ID", size=128)
     muskathlon_registration_ids = fields.One2many(
         'muskathlon.registration', 'event_id', 'Muskathlon registrations')
 
+    public_description = fields.Text('Description publique')
+    picture_1 = fields.Binary('Picture 1')
+    picture_2 = fields.Binary('Picture 2')
+    video_url = fields.Char("Video URL")
+    participants_amount_objective = fields.Integer(
+        'Default raise objective by participant', default=10000, require=True)
```

Comment:
the attribute is `required`. Please fix it everywhere you put `require`.

Reply:
Oup's sorry..